src/parameterized_tendencies/radiation/RRTMGPInterface.jl

```diff
@@ -901,6 +910,14 @@ function _RRTMGPModel(
         set_and_save!(z_lay, "center_z", t..., dict)
         z_lev = DA{FT}(undef, nlay + 1, ncol)
         set_and_save!(z_lev, "face_z", t..., dict)
+        if radiation_mode.deep_atmosphere && :planet_radius in keys(dict)
```

Comment:
```suggestion
if radiation_mode.deep_atmosphere 
    planet_radius = pop!(dict, :planet_radius, 6371000.0)
```

Reply:
or we can pick up the values from the `parameters` object.

Reply:
It would be much more robust, if surface area is stored, to take the area ratio of current level with the surface level!

Reply:
or have a functor similar to
```
function (dasf::DeepAtmosScalingFactor)(z_lev)
```

Reply:
These will be modified during the `RRTMGPInterface` updates shortly. I suggest we make that modification at that stage.